demo/index.html

```diff
@@ -23,6 +23,7 @@
       href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
     />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```

Comment:
Please update to mirror this code 

https://github.com/AlaskaAirlines/auro-avatar/blob/main/demo/index.html#L12-L17